docs/content/docs/flinkDev/ide_setup.md

```diff
@@ -260,7 +260,7 @@ versions.
 [Flake8](https://pypi.org/project/flake8/) is used to enforce some coding guidelines.
 
 1. Install flake8 for your Python interpreter using `pip install flake8`.
-2. In PyCharm go to "Preferences" → "Tools" → "External Tools".
+2. In PyCharm go to "Settings" → "Tools" → "External Tools".
```

Comment:
Could you please also update this in Chinese documents like [line 172](https://github.com/apache/flink/blob/master/docs/content.zh/docs/flinkDev/ide_setup.md?plain=1#L172) and [line 210](https://github.com/apache/flink/blob/master/docs/content.zh/docs/flinkDev/ide_setup.md?plain=1#L210)?

Reply:
hi, I am not fluent in Chinese, but I think line 172 is ok. fixing line 210 which discuss the Pycharm settings, incoming change.